docs/guides/scaffolding.md

```diff
@@ -9,7 +9,7 @@ In this guide, we will show how to:
 ## Setup
 
 !!!note
-    You might have already prepared your environment if you went through the [quick start](https://docs.autonolas.network/quick_start/) guide. If that's the case, you can skip this section.
+    You might have already prepared your environment if you went through the [quick start](https://docs.autonolas.network/guides/quick_start/) guide. If that's the case, you can skip this section.
```

Comment:
These links are all wrong @dvilelaf . You can see how I fixed them here: https://github.com/valory-xyz/open-autonomy/pull/1325